lib/teleterm/api/proto/v1/usage_events.proto

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.terminal.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gravitational/teleport/lib/teleterm/v1";
+
+message ReportEventRequest {
```

Comment:
As for importing connect.proto to service.proto:

> The protocol compiler searches for imported files in a set of directories specified on the protocol compiler command line using the -I/--proto_path flag. If no flag was given, it looks in the directory in which the compiler was invoked. In general you should set the --proto_path flag to the root of your project and use fully qualified names for all imports.

https://developers.google.com/protocol-buffers/docs/proto3#importing_definitions

AFAIK we now use `buf` instead of `protoc`, so maybe enabling that requires changes to buf's config.

Reply:
>Could the Electron app send prehog's events to tshd through TerminalService and then tshd would anonymize the required fields before passing the events further?

Unfortunately, it is not possible as tshd events (the ones that need to be anonymized) have one additional field - `auth_cluster_id`. It is needed to create the anonymizer. The prehog events don't have this field.

Reply:
I just pushed the changes, so we don't have tshd usage events anymore and there is much less boilerplate. 

We should merge changes to the protobuf generation first, @ravicious promised to create a PR for that :)